arch/xtensa/src/esp32s3/esp32s3_serial.c

```diff
@@ -445,9 +458,18 @@ static int esp32s3_setup(struct uart_dev_s *dev)
     }
 #endif
 
-  /* No Tx idle interval */
+#ifdef HAVE_RS485
+  if (priv->rs485_dir_gpio != 0)
+    {
+      esp32s3_lowputc_set_tx_idle_time(priv, 1);
```

Comment:
As an alternative we can go with value 1 for both RS485 and RS323 cases and remove additional if